pytorch_pretrained_bert/modeling_gpt2.py

```diff
@@ -617,8 +617,16 @@ def forward(self, input_ids, position_ids=None, token_type_ids=None, lm_labels=N
         hidden_states, presents = self.transformer(input_ids, position_ids, token_type_ids, past)
         lm_logits = self.lm_head(hidden_states)
         if lm_labels is not None:
+            # Shift so that tokens < n predict n
+            shift_logits = lm_logits[:, :-1].contiguous()
+            shift_labels = lm_labels[:, 1:].contiguous()
+
+            # In tensorflow, it's [batch, d_0, d_1, ..., d_{r-1}, num_classes]
```

Comment:
Can you remove this comment?
I don't think it's really related to the pytorch repo (so it may puzzle people reading the code).

Reply:
Yeah fair point. Helpful to me, but not the world.